File: programmers/문자열 계산하기.js
// my_string은 "3 + 5"처럼 문자열로 된 수식입니다. 문자열 my_string이 매개변수로 주어질 때, 수식을 계산한 값을 return

function solution(my_string) {
  let answer = 0;
  let array = my_string.split(" ");

  for (let i = 0; i < array.length / 2; i++) {
    console.log(array[2 * i - 1]);
    if ((array[2 * i - 1] === "+") | (array[2 * i - 1] === undefined)) {
      answer += Number(array[2 * i]);
    } else {
      answer -= Number(array[2 * i]);
    }
  }

  return answer;
}

console.log(solution("3 + 4"));

File: programmers/python/문자열 계산하기.py
# 문자열 계산하기
def solution(my_string):
    string = my_string.split()
    answer = int(string[0])
    for i in range(len(string) - 1):
        if i % 2 == 1:
            if string[i] == "+":
                answer += int(string[i + 1])
            else:
                answer -= int(string[i + 1])

    return answer